drivers/hmis/app/models/hmis/hud/processors/file_processor.rb

```diff
@@ -23,7 +23,8 @@ def process(field, value)
       elsif attribute_name == 'file_blob_id'
         return if attribute_value.nil?
 
-        blob = ActiveStorage::Blob.find_by(id: attribute_value)
+        # TODO(#7054) - accept either unsigned or signed blob ID for now, for backwards compatibility
+        blob = ActiveStorage::Blob.find_by(id: attribute_value) || ActiveStorage::Blob.find_signed(attribute_value)
```

Comment:
Maybe it doesn't matter, but should we prefer signed over unsigned?
```suggestion
        blob = ActiveStorage::Blob.find_signed(attribute_value) || ActiveStorage::Blob.find_by(id: attribute_value)
```